pmultiqc/cli.py

```diff
@@ -7,6 +7,7 @@
 http://click.pocoo.org/5/
 """
 
+from email.policy import default
```

Comment:
For what?

Reply:
The VScode editor mistakenly imported this automatically.  :sob: